Pokedex

maurperez opened the pull request.

Commit:
Obtencion datos y creacion de cada carta

asus753 commented:
![pokedex muestra](https://user-images.githubusercontent.com/56839325/73383828-d7c25e80-42a8-11ea-8557-32c6263f5ce8.png)

Commit:
Delete 25.png

Commit:
Reformulacion de la pagina

Cambios radicales al diseño de la pagina para evitar la utilizacion de tantos GET

Commit:
Presentacion de la pagina terminada completamente

La funcionalidad de la pagina ya esta casi completamente terminada, lo unico que queda es configurar la barra de busqueda

Commit:
Funcionalidad completamoente terminada

Agregamos manejo de errores y configuramos la barra de busqueda

Commit:
Actualizacion icono fire

asus753 commented:
![pokeapi](https://user-images.githubusercontent.com/56839325/74105201-73ba5880-4b3a-11ea-86a8-783906de416e.png)
![pokeapi2](https://user-images.githubusercontent.com/56839325/74105196-68ffc380-4b3a-11ea-8d1b-982ff5114fd8.png)

Commit:
Poniendo todo en ingles
